feat(curriculum): add chp3 md2 sharing personal details (#64546)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/es-a1-learn-basic-personal-information/6933347a77ffe4c5020e1cc9.md

---
id: 6933347a77ffe4c5020e1cc9
title: Task 12
challengeType: 19
dashedName: task-12
lang: es
---

<!-- (Audio) Ángela: Ah, de Perú. Muy bien. -->

# --description--

In this task, you will hear Ángela react to learning someone's country of origin.

# --instructions--

Listen to the audio and decide if the following statement is true or false.

# --questions--

## --text--

> Is it true or false: Ángela confirms the person is from Colombia.

Reviewer: ```suggestion
Is it true or false: Ángela confirms that Esteban is from Colombia.
```
I would suggest using Esteban's name for continuity.

## --answers--

True

### --feedback--

Listen again — Ángela does not mention Colombia.

---

False

## --video-solution--

2

# --explanation--

In the audio, Ángela clearly says `de Perú`, meaning "from Peru."  
She is confirming that the person is from Peru, **not** Colombia.

> # --scene--

Reviewer: ```suggestion
# --explanation--

In the audio, Ángela clearly says `de Perú`, meaning "from Peru."  Therefore, she is confirming that Esteban is from Peru, **not** Colombia.

# --scene--
```
